tests/pytest/test_fixtures.py

```diff
@@ -28,6 +32,26 @@ def infinity():
     return float('inf')
 
```

Comment:
Removed the extra tests - should just have 1 for the regular mock case and 1 for the autospecced mock case